android/build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
 
   dependencies {
     classpath "com.android.tools.build:gradle:7.2.2"
+    classpath 'com.google.gms:google-services:4.3.15'
```

Comment:
This is for FCM push messaging right? I don't think it belongs in the adapter itself, but in the app.